### PR TITLE
Updating events

### PR DESCRIPTION
**Pull request by giovp**
had 4 people across 2 continents saying that they couldn't find this anywhere on scverse webpage

### CODE REVIEW

**Comment by ivirshup** [comment]
> had 4 people across 2 continents saying that they couldn't find this anywhere on scverse webpage

WDYM? This specific page or the page Mark/ Nezar set up?

**Comment by giovp** [comment]
> > had 4 people across 2 continents saying that they couldn't find this anywhere on scverse webpage
> 
> WDYM? This specific page or the page Mark/ Nezar set up?

that the page from Mark/Nezar was not linked in our website.

File: static/img/events/conference_savethedate.jpeg
Comment (flying-sheep) [inline static/img/events/conference_savethedate.jpeg:1]: maybe also prefix the file name with `2024_09_`?

**Comment by emdann** [comment]
Added events for old hackathons & conference, but events in future dates still don't show. As suggested in zulip, it would probably make sense to change layout to split into Past and Upcoming events.

**Comment by gtca** [comment]
This should be working now! For all the future events, they will be automatically rendered either in the 'upcoming' or in the 'past' category depending on the date metadata field.

**Pull request merged**